realdecoy/rdvue-template
# Add frontier config file

**Summary of Changes**
Add support for frontier shorthand commands. The [frontier ](https://github.com/realdecoy/frontier)tool will automatically detect the project type based on the configuration in `.frontierrc` file and provide access to the commands based on the `vue` namespace in the CLI






**Related Issues**
N/A






**Additional Context**
N/A






**Screenshots/Videos**

![image](https://user-images.githubusercontent.com/53495324/229311392-57472ad3-8dba-4dd5-8938-5b0b25809402.png)






**Testing Steps**





1. clone this branch
2. install the `frontier` cli (beta)
   ```bash
   npm install --global @realdecoy/frontier@beta
   ```
3. run the following command and it should display the menu in the screenshot above
   ```bash
   frontier add
   ```
---



**Checklist**







-   [x] Sufficient unit tests added

-   [x] Sufficient logging added

-   [x] Sufficient in-line code documentation
